sklearn/cluster/_mean_shift.py

```diff
@@ -312,8 +314,10 @@ class MeanShift(ClusterMixin, BaseEstimator):
         If false, then orphans are given cluster label -1.
 
     n_jobs : int, default=None
-        The number of jobs to use for the computation. This works by computing
-        each of the n_init runs in parallel.
+        The number of jobs to use for the computation. following tasks benefit
+        from the parallelization: the search of nearest neighbors for bandwidth
+        estimation and label assignments, and the hill climbing optimization
+        for all seeds.
```

Comment:
Ditto
```suggestion
        The number of jobs to use for the computation. The following tasks benefit
        from the parallelization:
        - The search of nearest neighbors for bandwidth estimation and label assignments
        - Hill-climbing optimization for all seeds
        
        See :term:`Glossary <n_jobs>` for more details.
```

Reply:
Yes I'll add it

Reply:
With what we have to change I also feel it is better to write the para in points

Reply:
Yes I'll do it